cgp/cartesian_graph.py

```diff
@@ -263,46 +274,56 @@ def _format_output_str_numpy_of_all_nodes(self):
             for node in active_nodes[hidden_column_idx]:
                 node.format_output_str_numpy(self)
 
-    def to_numpy(self) -> Callable[[np.ndarray], np.ndarray]:
-        """Compile the function(s) represented by the graph to NumPy
-        expression(s).
+    def to_numpy(self) -> Callable[..., List[np.ndarray]]:
+        """Create a NumPy-array-compatible Python callable implementing the
+        function described by this graph.
 
-        Generates a definition of the function in Python code and
-        executes the function definition to create a Callable
-        accepting NumPy arrays.
+        The function expects as many arguments as the number of inputs
```

Comment:
`The returned function` or `The returned callable` to make it clear you are talking about the output and not this function?

Reply:
(actually similar as you did below for the torch instance)